assets/src/dashboard/dataUtils/formattedTemplatesArray.js

```diff
@@ -205,7 +205,7 @@ const formattedTemplatesArray = [
         backgroundOverlay: 'none',
         type: 'page',
         id: 'ID1',
-        pageLayoutType: 'cover',
+        pageTemplateType: 'cover',
```

Comment:
@BrittanyIRL This PR is changing `pageLayout` to `pageTemplate` everywhere (visibly + in code), found a few cases in the Dashboard, too. Let me know if there's something I might not be aware of and potentially breaking with this (seems to work ok when testing).

Reply:
thanks for flagging! i might not get to this today but at the very latest I will first thing monday.

Reply:
Thanks for checking. LMK if you found anything suspicious.

Reply:
Totally slipped off my radar, doing this ASAP, so sorry!